openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -266,6 +266,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
                 "Version": self.plugin_pype_version,
                 "Arguments": " ".join(args),
                 "SingleFrameOnly": "True",
+                "RenderSetupIncludeLights": instance.data.get("RenderSetupIncludeLights")  # noqa
```

Comment:
This key has no functionality on the submit publish job plug-in I think. This should be in Submit Maya Deadline, right? I feel like Deadline wouldn't get this value currently.

Reply:
Thank you @BigRoy I've done a couple of test renders seems like it goes through!

Reply:
> I've done a couple of test renders seems like it goes through!

I have a feeling that if it works that it's not necessarily due to this line of code but maybe due to it existing in the [payload skeleton in Submit Maya Deadline](https://github.com/pypeclub/OpenPype/blob/6dcd0ae47ed1baa2b5e11c97ce560948374bb499/openpype/modules/deadline/plugins/publish/submit_maya_deadline.py#L65) here.

Could you describe how you're detecting whether the value went through?

Are you able to see it on the Maya Render Job in deadline in **Modify Job Properties..** in `Submission Params` in `Plugin Info Parameters` as `RenderSetupIncludeLights=False`? And when the setting is toggled in Maya does it show `RenderSetupIncludeLights=True`?

Reply:
I'll elaborate, before adding the functionality to toggle the option on and off (which is actually the broken bit here), the flag was being propagated to the payload, yes, and it was going through into the submission params, but now that I'm trying to have it toggle the setting it's erroring out for me, which I'll remove from that commit i made and see if it works again.

Reply:
Hi Roy, you were right, for some reason I was remembering that this was working in that commit and I don't really understand why, it should indeed be in `Submit Maya Deadline` as part of the payload skeleton.  Now I can see the flag in there in the payload, testing for a layer.